styles/variables-silicon.less

```diff
@@ -182,6 +182,10 @@
 @agate-item-label-font-size:  30px;
 @agate-item-side-outset:      30px;
 @agate-item-sub-text-size:    24px;
+@agate-item-itemcontent-font-size:    33px;
+@agate-item-itemcontent-line-height:  36px;
+@agate-item-label-font-size:          24px;
+@agate-item-label-line-height:        27px;
```

Comment:
![image](https://user-images.githubusercontent.com/34676178/95310097-f44a7800-08c6-11eb-96ae-53cc24a36977.png)

Is there any reason you change line-height value?

Reply:
You're right. I just forgot to made the changes before pushing the commit. I changed them according to GUI. Thanks for noticing! 😅 
I also removed the `padding-top` property, I don't think it is necessary